Leanplum-SDK/Classes/Internal/Leanplum.m

```diff
@@ -1448,6 +1448,10 @@ + (void)onceVariablesChangedAndNoDownloadsPending:(LeanplumVariablesChangedBlock
     }
 }
 
++ (void)clearUserContent {
```

Comment:
You're creating this internally but never calling it. What is the use-case?

Reply:
Answered in the corresponding header file (which is public)